lib/index.js

```diff
@@ -45,6 +45,7 @@ var Amplitude = module.exports = integration('Amplitude')
   .option('saveParamsReferrerOncePerSession', true)
   .option('deviceIdFromUrlParam', false)
   .option('mapQueryParams', {})
+  .option('trackRevenuePerProduct', true)
```

Comment:
just to play devil's advocate, we sure we want this to default to `true`?

Reply:
Crap you're right, that should be false. Good catch